salesforce_functions/data_api/__init__.py

```diff
@@ -167,7 +168,7 @@ async def commit_unit_of_work(
         self, unit_of_work: UnitOfWork
     ) -> dict[ReferenceId, str]:
         """
-        Commit a `UnitOfWork`, executing all operations registered with it.
+        Commit a `UnitOfWork`, which executes all operations registered with it.
```

Comment:
IMO this is now ambiguous (does the commit execute all operations or does the `UnitOfWork`?)

Reply:
This is the wording change that they requested (and has already been made to the generated docs), and whilst I agree the "which" is ambiguous, I think it doesn't detract from the ability to understand what `commit_unit_of_work()` does, so I don't think it's worth another round trip between two repos to change further :-)